docs/modules/vearch.md

```diff
@@ -0,0 +1,57 @@
+# Vearch
+
+Since testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go/releases/tag/v0.29.0"><span class="tc-version">:material-tag: v0.29.0</span></a>
+
+## Introduction
+
+The Testcontainers module for Vearch.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the Vearch module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/vearch
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a Vearch container](../../modules/Vearch/examples_test.go) inside_block:runVearchContainer
```

Comment:
This path needs to be case sensitive, otherwise mkdocs will complain and not build the docs site:
```suggestion
[Creating a Vearch container](../../modules/vearch/examples_test.go) inside_block:runVearchContainer
```